mender/templates/gui-deploy.yaml

```diff
@@ -77,4 +77,8 @@ spec:
           value: {{ .Values.gui.env.GATEWAY_IP | quote }}
         - name: GATEWAY_PORT
           value: {{ .Values.gui.env.GATEWAY_PORT | quote }}
+        {{- if .Values.global.deviceconnect }}
+        - name: HAVE_DEVICECONNECT
+          value: {{ .Values.global.deviceconnect | quote }}
+        {{- end }}
```

Comment:
For these (both here and in workflows-worker) I don't know if I should use `Values.deviceconnect.enabled` instead of the new global that I have declared

Reply:
Both approaches are good; to be honest, I would have not introduced the "globals" and only used Values.deviceconnect.enabled. There is no point in having the deviceconnect service but not enabling it in the UI and in workflows...

Reply:
Makes sense. I originally just followed what was done for multitenancy, and then question it. I'll clean it up!